src/test/java/org/codehaus/plexus/components/io/attributes/FileAttributesTest.java

```diff
@@ -1,5 +1,7 @@
+package org.codehaus.plexus.components.io.attributes;
+
 /*
- * Copyright 2016 Plexus developers.
+ * Copyright 2011 The Codehaus Foundation.
```

Comment:
Actually `FileAttributesTest` is not modified at all. It's deleted and `Java7FileAttributesTest` is renamed to `FileAttributesTest` (the same goes for `FileAttributes` and `Java7FileAttributes`). Git show those as diffs but it's actually delete + rename. Maybe I should state that explicitly in the commit message as it's not obvious from looking at the diffs.